web/business-registry-dashboard/app/utils/affiliations.ts

```diff
@@ -197,21 +197,26 @@ export const isExpired = (item: Business, type?: CorpTypes): boolean => {
   if (type === CorpTypes.CONTINUATION_IN) {
     return affiliationStatus(item) === EntityStateStatus.APPROVED
   }
+  if (isDraft(affiliationStatus(item))) {
+    return (isIA(affiliationType(item)) || isAmalgamation(affiliationType(item)))
+  }
 
-  // Default behavior (IA/Registration/Amalgamation)
-  return isDraft(affiliationStatus(item)) && (isIA(affiliationType(item)) || isAmalgamation(affiliationType(item)))
+  return isExpiredDate
 }
 
 export const isExpiringSoon = (item: Business): { daysDiff: number, isSoon: boolean } => {
   // Return default if there's no expiration date
-  if (!item.nameRequest?.expirationDate) {
+  if (!item.nameRequest?.expirationDate || isExpired(item)) {
     return { daysDiff: NaN, isSoon: false }
   }
 
   const expirationDate = moment(item.nameRequest.expirationDate).tz('America/Vancouver')
   const currentDate = moment().tz('America/Vancouver')
 
-  const daysDiff = expirationDate.diff(currentDate.startOf('day'), 'days')
+  let daysDiff = expirationDate.diff(currentDate.startOf('day'), 'days')
+  if (daysDiff === 0 && expirationDate.isBefore(currentDate)) {
+    daysDiff = -1
+  }
```

Comment:
This feels clumsy. I think it would be better to update lines 213 and 214 to both be startOf('day'), and then simplify the diff on line 216. I tried that and it naturally returned -1 (or less) for expired NRs.